docs/source/command_line.rst

```diff
@@ -799,6 +801,30 @@ of the above sections.
         x = 'a string'
         x.trim()  # error: "str" has no attribute "trim"  [attr-defined]
 
+.. option:: --enable-all-error-codes
+
+    This flag enables all of the error codes for mypy,
+    including the optional ones that are off by default.
+    See :ref:`error-codes` and :ref:`error-codes-optional`
+    for more information.
+
+    This flag will override disabled error codes from the
```

Comment:
We should not do that. `--disable-error-code` should work as-is. Otherwise, it would be hard to use.

Reply:
Makes sense to me. I'll make the change.

I don't know why the overriding behavior for the extant *able-error-code flags is the way it currently is, anyway. Maybe just happenstance?